### DISCUSSION TITLE
数据布局服务与局部性管理 · OSDI 2018 - 面向信仰编程 · /papers-akkio

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#624] [Announcements]
https://draveness.me/papers-akkio/

**Comment by yuchanns** [comment]
这文章是不是延迟发布了。前几天来看还没看到。今天一看日期却写着8月11日

**Comment by draveness** [comment]
> 这文章是不是延迟发布了。前几天来看还没看到。今天一看日期却写着8月11日

只是日期填错了

**Comment by yuchanns** [comment]
> > 这文章是不是延迟发布了。前几天来看还没看到。今天一看日期却写着8月11日
> 
> 只是日期填错了

还是错了:joy:现在变成了0001-01-01
